### PR TITLE
[DOC] OAUTH authentication minor doc edit

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

### Type of change

**Documentation**

Minor doc edit to _Configuring OAuth 2.0 support for Kafka brokers_ section
Start of a change to treatment of replaceable text in guides (ongoing)

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/contributing/styleguide.adoc [hunk 1]
```diff
@@ -127,15 +127,15 @@ Learn more about reusing content in the link:https://redhat-documentation.github
 [[replaceables]]
 == Styling replaceables
 
-Replaceable values in the text and code must be styled as monospace, italics, lowercase, arrow brackets, and hyphens.
+Replaceable values in the text and code should be styled in `monospace`, with italics, uppercase, and hyphens.
```
Comment (scholzj) [inline documentation/contributing/styleguide.adoc:130]: Is this intentionaly in this PR? It seems to be unrelated so I'm not sure.

Comment (PaulRMellor) [reply documentation/contributing/styleguide.adoc:130]: It's half-related. We're starting to apply a new _replaceables_ format to the doc, which I used on the new oauth config. So I thought I would update the directions too.

File: documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc [hunk 2]
```diff
@@ -108,13 +108,13 @@ Depending on how you apply {oauth} authentication, and the type of authorization
     fallbackUserNameClaim: client_id <2>
     fallbackUserNamePrefix: client-account- <3>
     validTokenType: bearer <4>
-    userInfoEndpointUri: <__https://<auth-server-address>/auth/realms/external/protocol/openid-connect/userinfo__> <5>
+    userInfoEndpointUri: __https://OAUTH-SEVER-ADDRESS/auth/realms/external/protocol/openid-connect/userinfo__ <5>
```
Comment (mstruk) [inline documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc:111]:
```suggestion
    userInfoEndpointUri: __https://OAUTH-SERVER-ADDRESS/auth/realms/external/protocol/openid-connect/userinfo__ <5>
```

File: documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc [hunk 3]
```diff
@@ -108,13 +108,13 @@ Depending on how you apply {oauth} authentication, and the type of authorization
...
-<5> (Only applicable when using `introspectionEndpointUri`) The authorization server may be configured or implemented in such a way to not provide any identifiable information in an Introspection Endpoint response. In order to obtain the user id, you can configure the URI of the User Info Endpoint as a fallback. The `userNameClaim`, `fallbackUserNameClaim`, and `fallbackUserNamePrefix` settings are applied to the response of User Info Endpoint.
+<2> An authorization server may not provide a single attribute to identify both regular users and clients. A client authenticating in its own name might provide a _client ID_. But a user authenticating using a username and password, to obtain a refresh token or an access token, might provide a _username_ attribute in addition to a client ID. Use this fallback option to specify the username claim (attribute) to use if a primary user ID attribute is not available.
```
Comment (mstruk) [inline documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc:114]: I realise this is a copy paste from upstream docs, and reading it again I think that it should be improved.

```suggestion
<2> An authorization server may not provide a single attribute to identify both regular users and clients. When a client authenticates in its own name the server might provide a _client ID_. When a user authenticates using a username and password, to obtain a refresh token or an access token, the server might provide a _username_ attribute in addition to a client ID. Use this fallback option to specify the username claim (attribute) to use if a primary user ID attribute is not available.
```

File: documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc [hunk 4]
```diff
@@ -108,13 +108,13 @@ Depending on how you apply {oauth} authentication, and the type of authorization
...
+<3> In situations where `fallbackUserNameClaim` is applicable, it may also be necessary to prevent name collisions between the values of the username claim, and those of the fallback username claim. Consider a situation where a client called `producer` exists, but also a regular user called `producer` exists. In order to differentiate between the two, you can use this property to add a prefix to the user ID of the client.
+<4> (Only applicable when using `introspectionEndpointUri`) Depending on the authorization server you are using, the introspection endpoint may or may not return the _toke type_ attribute, or it may contain different values. You can specify a valid token type value that the response from the introspection endpoint has to contain.
```
Comment (mstruk) [inline documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc:116]: I'm not sure if this was a typo - `token_type` is the literal name of the attribute returned buy the server. It seems as if you wanted to remove the underscore, but you mistakenly removed the letter 'n'. Was that your intent?
 
```suggestion
<4> (Only applicable when using `introspectionEndpointUri`) Depending on the authorization server you are using, the introspection endpoint may or may not return the _token type_ attribute, or it may contain different values. You can specify a valid token type value that the response from the introspection endpoint has to contain.
```

File: documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc (hunk 2 above)
Comment (PaulRMellor) [reply documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc:111]: Typo!

File: documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc (hunk 4 above)
Comment (PaulRMellor) [reply documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc:116]: Typo!
